src/subcommand/add.rs

```diff
@@ -31,7 +31,14 @@ impl Add {
 
 fn add<P: AsRef<Path>>(path: P) -> Result<()> {
     let path = path.as_ref();
-    let path = canonicalize(&path)?;
+    let current_dir;
+    let path = match config::zo_resolve_symlinks() {
+        true => canonicalize(&path)?,
```

Comment:
if-else might be clearer than matching against boolean values.

Reply:
Done, I didn't know you could use if/else for variable assignation.